components/dropdown/Dropdown.module.css

```diff
@@ -27,8 +29,8 @@
   background-color: transparent;
   cursor: pointer;
   position: absolute;
-  top: 99px;
-  left: 313px;
```

Comment:
So instead of huge values left / top we just have 11px top and 12px right (so it's on the right side of the dropdown in vertically centered)